app/src/main/java/net/gsantner/markor/frontend/textview/TextViewUtils.java

```diff
@@ -567,27 +600,27 @@ public InputFilter[] getFilters() {
 
         @Override
         public void setSpan(Object what, int start, int end, int flags) {
-            // Do nothing
+            setSel(what, start);
```

Comment:
Very limited span support so that we can set and get selection from a ChunkedEditable.